yoti-sdk-impl/src/test/java/com/yoti/api/client/shareurl/policy/SimpleDynamicPolicyBuilderTest.java

```diff
@@ -192,6 +193,64 @@ public void buildWithRememberMeFlags() {
         assertTrue(result.isWantedRememberMeOptional());
     }
 
+    @Test
+    public void buildsWithSelfieAuthorisationEnabledThenDisabled() {
+        DynamicPolicy result = new SimpleDynamicPolicyBuilder()
+                .withSelfieAuthorisation(true)
+                .withSelfieAuthorisation(false)
+                .build();
+
+        assertThat(result.getWantedAuthTypes(), not(hasItem(1)));
+    }
+
+    @Test
+    public void buildsWithSelfieAuthorisationDisabledThenEnabled() {
+        DynamicPolicy result = new SimpleDynamicPolicyBuilder()
+                .withSelfieAuthorisation(false)
+                .withSelfieAuthorisation(true)
+                .build();
+
+        assertThat(result.getWantedAuthTypes(), hasItem(1));
+    }
+
+    @Test
+    public void buildsWithPinAuthorisationEnabledThenDisabled() {
+        DynamicPolicy result = new SimpleDynamicPolicyBuilder()
+                .withPinAuthorisation(true)
+                .withPinAuthorisation(false)
+                .build();
+
+        assertThat(result.getWantedAuthTypes(), not(hasItem(2)));
+    }
+
+    @Test
+    public void buildsWithPinAuthorisationDisabledThenEnabled() {
+        DynamicPolicy result = new SimpleDynamicPolicyBuilder()
+                .withPinAuthorisation(false)
+                .withPinAuthorisation(true)
+                .build();
+
+        assertThat(result.getWantedAuthTypes(), hasItem(2));
+    }
+
+    @Test
+    public void buildsWithSelfieAuthorisationDisabled() {
+        DynamicPolicy result = new SimpleDynamicPolicyBuilder()
+                .withSelfieAuthorisation(false)
+                .build();
+
+        assertThat(result.getWantedAuthTypes(), hasItem(1));
```

Comment:
Should this be `not(hasItem(1))`?

Reply:
Once we add Travis tests, these can automatically be picked up 😃 👍